src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/ItemRoute.js

```diff
@@ -38,6 +37,14 @@ export const ItemRoute = connect((state) => {
 
         return dispatch(searchItems(fullPath))
           .then((res) => {
+            if (!res) {
```

Comment:
The lack of a `res` would indicate the code failed, not that there was not a path.